core/server/app.js

```diff
@@ -22,6 +23,7 @@ module.exports = function setupParentApp() {
     // (X-Forwarded-Proto header will be checked, if present)
     parentApp.enable('trust proxy');
 
+    parentApp.use(requestId);
```

Comment:
This does not work, because the user is loaded later. It worked in the log request middleware, because it access the req object on response finish event.